tests/conftest.py

```diff
@@ -8,44 +8,32 @@ def pytest_configure():
     from django.conf import settings
 
     settings.configure(
-        SECRET_KEY="secret",
-        DEBUG=True,
```

Comment:
Fun. This was never actually true. Seems `configure` cannot override the default of `False` from `global_settings`.